routing/index_graph_starter.cpp

```diff
@@ -43,14 +46,36 @@ m2::PointD const & IndexGraphStarter::GetPoint(Segment const & segment, bool fro
 }
 
 // static
-size_t IndexGraphStarter::GetRouteNumPoints(vector<Segment> const & segments)
+void IndexGraphStarter::CheckValidRoute(vector<Segment> const &segments)
```

Comment:
From my point of view, valid route may contain even no segments at all, if start point is the same as the end point.